hack/run-e2e.sh

```diff
@@ -174,9 +174,10 @@ function e2e::setup_helm_server() {
     if hack::version_ge $(e2e::get_kube_version) "v1.16.0"; then
         # workaround for https://github.com/helm/helm/issues/6374
         # TODO remove this when we can upgrade to helm 2.15+, see https://github.com/helm/helm/pull/6462
+        # \'$'\n  regex is used to be compatible with BSD sed (Darwin)
```

Comment:
```suggestion
        # \'$'\n is used to be compatible with BSD sed (Darwin)
```

it's not regex actually.

Reply:
ok